crates/router/src/connector/adyen/transformers.rs

```diff
@@ -266,6 +269,16 @@ pub struct RedirectionResponse {
     refusal_reason_code: Option<String>,
 }
 
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AdyenPTSResponse {
```

Comment:
Nit: naming convention: would prefer `AdyenPtsResponse` and `AdyenPtsAction` instead.

Reply:
Done👍